app/models/quiz.rb

```diff
@@ -14,6 +14,41 @@ def answer(answer_text:)
         answers.create(quiz_id: self.quiz_id,answer_text: answer_text,answer_succeed: self.is_answer_succeed?(answer_text))
     end
 
+    def create_question_message
+        now_quiz_pokemon_id = quiz.where(user_id: user_id).last.pokemon_id
+
+        question_message = {
+            type: 'text',
+            text: '問題！このポケモンはなんでしょう？ '+now_quiz_pokemon_id
+        }
```

Comment:
[IMHO]
インスタンスメソッドなので、クイズは特定されている前提だと思います。
改めて Quiz を特定する必要は無さそうですがどうでしょう？（`pokemon_id` でクイズに設定されているポケモンIDは取れそう）

※ `create_image_message` に関しても同様

```suggestion
        question_message = {
            type: 'text',
            text: '問題！このポケモンはなんでしょう？ '+ pokemon_id
        }
```

Reply:
[7baa962](https://github.com/Ishigami100/intern-line-bot/pull/4/commits/7baa962575729cd676cd88397e61ba77adb2c2d3)→こちらでpokemon_idで取れるように直しました。